tests/integration/snapshots/600.framework-detection.test.js.md

```diff
@@ -242,4 +242,4 @@ Generated by [AVA](https://avajs.dev).
     > node -p process.env.GATSBY_LOGGER␊
     ␊
     yurnalist␊
-    ◈ "npm run develop" exited with code *. Shutting down Netlify Dev server`
+    .◈ "npm run develop" exited with code *. Shutting down Netlify Dev server`
```

Comment:
What is this about?

Reply:
A flaky test that I was trying to get around, there is a helper function that concats a series of loading dots .... down to a single dot.  But occasionally the load happens fast enough to not have any dots at all (at least that is my theory).  I was toying with the regex in the helper but gave up on it.  This change was actually an error, since it's an autogenerated file anyway, and isn't going to help me pass the test at all.

Reply:
Not necessary, I have another PR going to fix that test

Reply:
Okay, but why would we merge this change? We should revert and let the other PR fix the test?

Reply:
Ok, that makes sense.  Sorry, I'm still in weekend mode!